S21/Rongxiang-Zhang/lower.py

```diff
@@ -0,0 +1,60 @@
+import csv
+
+
+def wordlower(word,lan):
+
+    word=word
+
+    lan=lan
+
+    if(lan=='en'):
+        for letter in word:
+            letter=word.lower()
+        print(letter)
+
+
+    if(lan=='tr' or lan=='az'):
+
+        for letter in word:
+            if (letter=='i' or letter=='I'):
+                letter=word.replace('I',"ı").lower()
+                print(letter)
+            else:
+                print(letter.lower(),end='')
+
+
+    if(lan=='ga'):
+        leng=len(word)
+
+        if(word[0]=='n' or word[0]=='t'):
+            if(word[1]==a or word[1]==E or word[1]==I or word[1]==O or word[1]==U or word[1]==Á or word[1]==Í or word[1]==Ó or word[1]==Ú):
```

Comment:
This if statement is long, we could put all the vowels in a list. We can say if word[1] in list of vowels, run this logic

Reply:
I wanted to give a variable to every vowel, but it seemed too long.